docs/user-guide/command-line-interface.md

```diff
@@ -4,23 +4,23 @@ To run ESLint on Node.js, you must have npm installed. If npm is not installed,
 
 Once npm is installed, run the following
 
-    npm i -g eslint
+    npm i eslint --save-dev
```

Comment:
Would it make sense to just link to https://eslint.org/docs/user-guide/getting-started#installation-and-usage here? User has to set up a configuration file after installing ESLint, which isn't mentioned at all in this document.

Reply:
Good idea!